sidekick/sidekick-phone-home/src/main/java/com/runsidekick/model/EventType.java

```diff
@@ -0,0 +1,22 @@
+package com.runsidekick.model;
+
+/**
+ * @author yasin.kalafat
+ */
+public enum EventType {
+    SERVER_UP("SERVER_UP");
+
+    private String text;
+
+    EventType(String text) {
+        this.text = text;
+    }
+
+    public String getText() {
+        return text;
+    }
+
+    public void setText(String text) {
```

Comment:
Since this is enum, properties should not be modified. So it is better to remove `setText`